src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py

```diff
@@ -618,8 +618,7 @@ def __call__(
 
 
 # resizing utils
-# TODO: clean up later
-def _resize_with_antialiasing(input, size, interpolation="bicubic", align_corners=True):
```

Comment:
This utility is private so, probably okay to not touch it?

Reply:
Yes, I agree that it is okay to not touch it, but I also think that it is better to clean up even private utilities. Although it is written in TODO that it needs to be cleaned.